src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimantResponseService.java

```diff
@@ -92,9 +93,12 @@ private boolean isReferredToJudge(ClaimantResponse response) {
 
     private boolean isRejectPartAdmitNoMediation(ClaimantResponse claimantResponse, Claim claim) {
         Response response = claim.getResponse().orElseThrow(IllegalStateException::new);
+
         return ResponseType.PART_ADMISSION.equals(response.getResponseType())
             && ClaimantResponseType.REJECTION.equals(claimantResponse.getType())
-            && !((ResponseRejection) claimantResponse).getFreeMediation().orElse(false);
+            && ((ResponseRejection) claimantResponse).getFreeMediation()
+            .filter(Predicate.isEqual(YesNoOption.NO))
+            .isPresent();
```

Comment:
How about?
```
NO == ((ResponseRejection) claimantResponse).getFreeMediation().orElse(NO)
```

Reply:
May be leave it like that, looks more java 8ishh 😄

Reply:
I suppose better would be `        return ResponseType.PART_ADMISSION.equals(response.getResponseType())
            && ClaimantResponseType.REJECTION.equals(claimantResponse.getType())
            && ((ResponseRejection) claimantResponse).getFreeMediation().orElse(YesNoOption.NO) != YesNoOption.YES;`